src/khoj/processor/conversation/utils.py

```diff
@@ -1,35 +1,19 @@
 # Standard Packages
-import os
 import logging
-from datetime import datetime
 from time import perf_counter
-from typing import Any
-from threading import Thread
 import json
+from datetime import datetime
+import tiktoken
 
-# External Packages
-from langchain.chat_models import ChatOpenAI
+# External packages
 from langchain.schema import ChatMessage
-from langchain.callbacks.streaming_stdout import StreamingStdOutCallbackHandler
-from langchain.callbacks.base import BaseCallbackManager
-import openai
-import tiktoken
-from tenacity import (
-    before_sleep_log,
-    retry,
-    retry_if_exception_type,
-    stop_after_attempt,
-    wait_exponential,
-    wait_random_exponential,
-)
 
 # Internal Packages
-from khoj.utils.helpers import merge_dicts
 import queue
-
+from khoj.utils.helpers import merge_dicts
 
 logger = logging.getLogger(__name__)
-max_prompt_size = {"gpt-3.5-turbo": 4096, "gpt-4": 8192}
+max_prompt_size = {"gpt-3.5-turbo": 4096, "gpt-4": 8192, "text-davinci-001": 910}
```

Comment:
why not call use "falcon" as the key for the max_prompt size for falcon?
```suggestion
max_prompt_size = {"gpt-3.5-turbo": 4096, "gpt-4": 8192, "falcon": 910}
```

Reply:
The `tiktoken` library has a specific set of models it works with. I just used this as a hack so we can specify a single model name when doing prompt truncation with Falcon.